podcasts/PodcastListViewController+Tip.swift

```diff
@@ -0,0 +1,78 @@
+import SwiftUI
+import PocketCastsUtils
+
+extension PodcastListViewController: UIPopoverPresentationControllerDelegate {
+    func showRecentlyPlayedSortingTipIfNeeded() {
+        guard
+            Settings.shouldShowRecentlyPlayedSortingTip,
+            FeatureFlag.podcastsSortChanges.enabled,
+            recentlyPlayedSortingTip == nil
+        else {
+            return
+        }
+        if let vc = showRecentlyPlayedSortingTip() {
+            present(vc, animated: true) {
+                Analytics.track(.episodeRecentlyPlayedSortOptionTooltipShown)
+            }
+            recentlyPlayedSortingTip = vc
+        }
+    }
+
+    private func showRecentlyPlayedSortingTip() -> UIViewController? {
+        guard let button = customRightBtn else {
+            return nil
+        }
+        let vc = UIHostingController(rootView: AnyView (EmptyView()) )
+        let idealSize = CGSizeMake(290, 100)
```

Comment:
⛏️ Could use the swiftier `CGSize` here:
```suggestion
        let idealSize = CGSize(290, 100)
```